src/app/card/info-status-card/info-status-card.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="card-pf card-pf-info-status" [class.card-pf-accented]="config.showTopBorder">
+  <div class="card-pf-info-image">
+    <img *ngIf="config.iconImageSrc" src="{{config.iconImageSrc}}" alt="" class="info-img">
+    <span class="info-icon {{config?.iconStyleClass}}"></span>
```

Comment:
FYI, you really don't need '?' for config?.iconStyleClass because you ensure config is created via the component. I only suggested using it because you had something more like config.status?.iconClass, where status could be undefined.